include/deal.II/base/time_stepping.templates.h

```diff
@@ -241,6 +241,232 @@ namespace TimeStepping
+                        393957816125. / 7825732611452.,
+                        720647959663. / 6565743875477.,
+                        3559252274877. / 14424734981077.}};
+            std::vector<double> ai;
+            ai = {{1107026461565. / 5417078080134.,
+                   38141181049399. / 41724347789894.,
+                   493273079041. / 11940823631197.,
+                   1851571280403. / 6147804934346.,
+                   11782306865191. / 62590030070788.,
+                   9452544825720. / 13648368537481.,
+                   4435885630781. / 26285702406235.,
+                   2357909744247. / 11371140753790.}};
+            this->a.push_back(ai);
+            break;
+          }
+        default:
+          {
+            AssertThrow(false,
+                        ExcMessage(
+                          "Unimplemented low-storage Runge-Kutta method."));
+          }
+      }
+    // compute ci
+    this->c.reserve(this->n_stages);
+    this->c.push_back(0.);
+    double sum_previous_bi = 0.;
+    for (unsigned int stage = 1; stage < this->n_stages; ++stage)
+      {
+        const double tmp = sum_previous_bi + this->a[0][stage - 1];
+        this->c.push_back(tmp);
+        sum_previous_bi += this->b[stage - 1];
+      }
+  }
+
+
+
+  template <typename VectorType>
+  double
+  LowStorageRungeKutta<VectorType>::evolve_one_time_step(
+    const std::function<VectorType(const double, const VectorType &)> &f,
+    const std::function<
+      VectorType(const double, const double, const VectorType &)>
+      & /*id_minus_tau_J_inverse*/,
+    double      t,
+    double      delta_t,
+    VectorType &y)
+  {
+    VectorType vec_ri;
+    VectorType vec_ki;
+    return evolve_one_time_step(f, t, delta_t, y, vec_ri, vec_ki);
+  }
+
+
+
+  template <typename VectorType>
+  double
+  LowStorageRungeKutta<VectorType>::evolve_one_time_step(
+    const std::function<VectorType(const double, const VectorType &)> &f,
+    double                                                             t,
+    double                                                             delta_t,
+    VectorType &                                                       solution,
+    VectorType &                                                       vec_ri,
+    VectorType &                                                       vec_ki)
+  {
+    // From step-67
+    compute_one_stage(f,
+                      t,
+                      this->b[0] * delta_t,
+                      this->a[0][0] * delta_t,
+                      solution,
+                      vec_ki,
+                      solution,
+                      vec_ri);
+
+    for (unsigned int stage = 1; stage < this->n_stages; ++stage)
+      {
+        const double c_i = this->c[stage];
+        compute_one_stage(f,
+                          t + c_i * delta_t,
+                          this->b[stage] * delta_t,
+                          (stage == this->n_stages - 1 ?
```

Comment:
can you compute this constant in the line above instead and store in a "const double"?